pkg/apis/dashboard/v0alpha1/types.go

```diff
@@ -112,3 +110,13 @@ type AnnotationActions struct {
 	CanEdit   bool `json:"canEdit"`
 	CanDelete bool `json:"canDelete"`
 }
+
+type Spec struct {
```

Comment:
um... this is not a dashboard 😬 

The unstructured one holds on to everything, so is still a dashboard, but obviously we need to make it better

Reply:
> um... this is not a dashboard

Yes, I know, I'm just trying to get things working so that we can try adding access-control checks to the unified storage. (I'm not planning to merge this to main)

If you want, let's catch up on the dashboard issue later on :pray: